rca/projects/models.py

```diff
@@ -19,7 +22,7 @@
 from wagtail.images import get_image_model_string
 from wagtail.images.edit_handlers import ImageChooserPanel
 
-from rca.home.models import HERO_COLOUR_CHOICES, LIGHT_TEXT_ON_DARK_IMAGE
+from rca.home.models import HERO_COLOUR_CHOICES, LIGHT_HERO, LIGHT_TEXT_ON_DARK_IMAGE
```

Comment:
I've added a separate issue for what we spoke about here https://github.com/torchbox/rca-wagtail-2019/issues/395